Core/CoreTests/Login/LoginStartViewControllerTests.swift

```diff
@@ -179,23 +180,34 @@ class LoginStartViewControllerTests: CoreTestCase {
 
     func testQRLoginFeatureGetsTurnedOn() {
         supportsQRCodeLogin = true
-        ExperimentalFeature.qrLogin.isEnabled = false
         controller.view.layoutIfNeeded()
+        controller.viewDidLoad()
         XCTAssertTrue(controller.useQRCodeButton.isHidden)
         XCTAssertTrue(controller.useQRCodeDivider.isHidden)
-        ExperimentalFeature.qrLogin.isEnabled = true
-        XCTAssertFalse(controller.useQRCodeButton.isHidden)
-        XCTAssertFalse(controller.useQRCodeDivider.isHidden)
     }
 
     func testQRLoginFeatureGetsTurnedOff() {
         supportsQRCodeLogin = true
-        ExperimentalFeature.qrLogin.isEnabled = true
         controller.view.layoutIfNeeded()
+        controller.viewDidLoad()
         XCTAssertFalse(controller.useQRCodeButton.isHidden)
         XCTAssertFalse(controller.useQRCodeDivider.isHidden)
-        ExperimentalFeature.qrLogin.isEnabled = false
-        XCTAssertTrue(controller.useQRCodeButton.isHidden)
```

Comment:
Similar to the test above, this test is meant to assert what happens when the feature flag changes. Now the assertions don't match the test name.